src/libs/ReportUtils.ts

```diff
@@ -8241,9 +8241,7 @@ function createDraftTransactionAndNavigateToParticipantSelector(transactionID: s
         mccGroup,
     } as Transaction);
 
-    const filteredPolicies = Object.values(allPolicies ?? {}).filter(
-        (policy) => policy && policy.type !== CONST.POLICY.TYPE.PERSONAL && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
-    );
+    const filteredPolicies = PolicyUtils.getActivePolicies(allPolicies, currentUserEmail, true);
```

Comment:
Let's revert this; it's not going to result in the exact same logic so could introduce regressions.

Reply:
Oops sorry all the lint fixes got to me :smile: